### ISSUE TITLE
hyperparameters issue

### ISSUE THREAD

**Issue by domenicodegioia** [tanatosuu/GDE#6]
how about best hyperparameters on yelp dataset?

**Comment by tanatosuu** [comment]
Thanks for your interests in our work. It depends on which model you refer. If it is GDE (SIGIR '22) which we haven't tested on Yelp, can you strat with initial setting like: drop_out=0.1, smooth_ratio=0.1~0.2 (preprocess), beta=4.0~5.0, and adjust the paramater a little bit to determine the best setting? (A large drop out raio could cause a 'nan' loss issue)

If you refer to the Journal (TOIS) version, according to my record:

alpha=3, req_vec=60 for preprocess

beta=2.0, coef_u=0.7, coef_i=0.7, std=0.025, learning_rate=7.0

on the Yelp dataset used in the paper (Changing the training/test ratio or using another Yelp data could also cause the change)

I haven't run the code for a while. If you have any problems on the codes, please don't hesitate to contact me.

**Comment by domenicodegioia** [comment]
thank you so much for the reply

is it correct to say that this set of hyperparameters ( https://github.com/tanatosuu/GDE/blob/main/datasets/parameter_settings.txt ) can be used for replicating results of the SIGIR '22 paper?

**Comment by domenicodegioia** [comment]
thank you so much for the reply

is it correct to say that this set of hyperparameters ( https://github.com/tanatosuu/GDE/blob/main/datasets/parameter_settings.txt ) can be used for replicating results of the SIGIR '22 paper?

**Comment by tanatosuu** [comment]
Yes. They are exactly the hyperparameter settings for the SIGIR '22 paper on the datasets used in the paper. However, the results somehow have randomness to some extent, i.e., change with the environment (GPU, CPU, etc.). If you encounter any issues running the model, please feel free to contact me.

**Comment by domenicodegioia** [comment]
I was able to reproduce the results of the SIGIR '22 article.

Thank you very much for your help.
